tests/driver/nodejs/Cargo.toml

```diff
@@ -14,12 +14,12 @@ publish = false
 
 [[bin]]
 path = "main.rs"
-name = "test-driver-nodejs"
+name = "test-driver-napi"
```

Comment:
Further up in the diff you changed the CI instructions to run this command:

```
run: cargo test --verbose --all-features -p test-driver-nodejs -p slint-node
```

(used to be `-p test-driver-napi`)

Why do you rename the binary to not match the package name?

Reply:
This was a mistake and is fixed now.